src/components/Protein/Isoforms/Viewer/index.tsx

```diff
@@ -70,18 +70,45 @@ const features2protvista = (features: FeatureMap) => {
   ];
 
   const sortedCategories = categories.sort(byEntryType);
+
   const representativeDomains = selectRepresentativeData(
     featArray,
     'locations',
-    'domain'
+    'domain',
   );
 
-  if (representativeDomains?.length) {
-    sortedCategories.splice(0, 0, [
-      'representative domains',
-      representativeDomains,
-    ]);
-  }
+  const representativeFamilies = selectRepresentativeData(
+    featArray,
+    'locations',
+    'family',
+  );
+
+  const homologous_superfamily = sortedCategories.filter(
+    (entry) => entry[0] == 'homologous superfamily',
+  )[0];
+
+  sortedCategories.map((entry) => {
+    if (entry[0] === 'domain') {
```

Comment:
If this is the type (as returned by the API and found in `metadata.type`), then the type _repeat_ should be added (representative domains are for signatures of type domain _and_ repeat).